platform/marvell-armhf/platform.conf

```diff
@@ -149,9 +191,53 @@ create_ubi_partition() {
     ubimkvol  /dev/$demo_dev -N $demo_dev -s 3900MiB
 }
 
+create_gpt_partition() {
+    blk_dev="/dev/sda"
+    demo_part=$(sgdisk -p $blk_dev | grep -e "$demo_volume_label" -e "$legacy_volume_label" | awk '{print $1}')
+    # ONIE partition size 168MB
+    onie_part_size=168
```

Comment:
>168 [](start = 19, length = 3)

How do you determine this number?

Reply:
If board is from manufacturing without sonic partition, then skip onie partition size as per platform partition layout